CRCSim/R/compare.R

```diff
@@ -25,11 +25,11 @@ compare <- function(sim, groundTruth) {
     }
     out <- rbindlist(compare.list)
   } else {
-    aic <- sim$AIC
-    est <- sim$estimate
-    est.lci <- sim$lower_ci
-    est.uci <- sim$upper_ci
-    form <- sim$formula
+    aic <- sim[[1]]$AIC
+    est <- sim[[1]]$estimate
+    est.lci <- sim[[1]]$lower_ci
+    est.uci <- sim[[1]]$upper_ci
+    form <- sim[[1]]$formula
```

Comment:
Is sim guaranteed to contain data or is this a potential error point if sim is empty?

Reply:
Sim is guaranteed to be a list of 1 in this section. In the even sim is not returned this will produce NAs